crates/router/src/core/payments/operations/payment_confirm.rs

```diff
@@ -922,12 +922,12 @@ impl<F: Clone + Send> Domain<F, api::PaymentsRequest, PaymentData<F>> for Paymen
             Some(helpers::PaymentExternalAuthenticationFlow::PostAuthenticationFlow {
                 authentication_id,
             }) => {
-                let authentication = authentication::perform_post_authentication(
+                let authentication = Box::pin(authentication::perform_post_authentication(
```

Comment:
Why is the `Box::pin` required `clippy::large_futures`??

Reply:
yep it was failing in compilation.